scripts/create-component.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+process.on('exit', code => {
+  if(code === 0) {
+    console.log(`New component is created successfully.`)
+  }
+});
+
+const path = require('path');
+const fs = require('fs');
+const fileSave = require('file-save');
+const uppercamelcase = require('uppercamelcase');
+const glob = require('glob');
+const ATOMIC_TYPE = [
+  'atoms',
+  'molecules',
+  'organisms'
+]
+
+if(process.argv.length < 4) {
+  console.warn('Please input atomic type and component name')
+  process.exit(-1);
+}
+
+createComponent(process.argv[2],process.argv[3])
+
+
+function createComponent(folder, componentName ) {
+  if(!ATOMIC_TYPE.includes(folder)) {
+    console.warn('Following atomic types are supported: atoms, molecules, organisms')
+    process.exit(-1);
+  }
+  const ComponentName = uppercamelcase(componentName);
+  const PrefixComponentName = 'Sf' + ComponentName;
```

Comment:
You're defining that component name from args should not have _Sf_ prefix, right? Wdyt about making it optional?

```js
const PrefixComponentName = ComponentName.startsWith('Sf') ? ComponentName : 'Sf' + ComponentName;
```

Reply:
You are right. It's more comprehensive. I fixed it.